jenkins-files/planetway/artifacts/tools/publish_crl_to_s3.py

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env python3
+
+import boto3
+import logging
+import sys
+import os
+from botocore.exceptions import ClientError, NoCredentialsError
+from cryptography import x509
+
+
+def file_name_from_crl(crl):
```

Comment:
I don't know the context but, I'd make this python script minimal, just write the filename to stdout and pipe that to awscli to upload to S3.